starter-code/app.js

```diff
@@ -11,7 +11,7 @@ const path         = require('path');
 
 
 mongoose
-  .connect('mongodb://localhost/starter-code', {useNewUrlParser: true})
+  .connect('mongodb://localhost/celebrities', {useNewUrlParser: true})
   .then(x => {
     console.log(`Connected to Mongo! Database name: "${x.connections[0].name}"`)
   })
```

Comment:
youre unecessarily repeating this code.

You could have just done something liek:

```js
require("./configs/db.config")
```

And you wouldnt have duplicated code in your project